src/refmvs.rs

```diff
@@ -54,6 +56,7 @@ impl From<[i8; 2]> for RefMvsRefPair {
 pub struct RefMvsMvPair {
     pub mv: [Mv; 2],
 }
+const _: () = assert!(mem::size_of::<RefMvsTemporalBlock>() == 5);
 
 #[derive(Clone, Copy, FromZeroes)]
 #[repr(C, align(4))]
```

Comment:
We already have the `RefMvsBlock` size check below, but could you move the comment to the type now.  Previously, I had added the size check in Rust (it wasn't in C yet) since I didn't pack it, since Rust doesn't let you, so we had to manually ensure it was packed.